### DISCUSSION TITLE
Flare explanation?

### DISCUSSION THREAD

**Discussion by PeanutCultist** [TEHE-Studios/ExpandedHelicopterEvents#172] [Q&A] [answered]
Hi there, I know a similar question was asked previously in a different discussion, but I still have a few questions regarding them.

What is the difference between "Flare" and "Signal Flare"?
If I see the AEB say something about activity in a different sector, or after the broadcast when it specifies an event, will popping a flare cause it to redirect to the flare's location? If so, in the case of supply drops, how close to the flare?

**Comment by Chuckleberry-Finn** [comment, marked as answer]
Signal flares fire into the sky, the other type of flare is held in the hand.

Some events are set to redirect to more appealing targets, the range of this is usually within in the cell.

**Reply by PeanutCultist** [reply]
Thanks for the response! This raises another question for me, though.
What do you mean by "fire into the sky"? I've been leaving them on the ground after I light them, am I supposed to use some kind of flare gun?
Also, can you specify the events that can be redirected? Thanks again in advance.

**Reply by Chuckleberry-Finn** [reply]
If the flare is "spent" right away the light will stay in place and the item in hand is dead. The other type of flare will light and you can carry it around.

**Comment by Chuckleberry-Finn** [comment]
#131 Linking other discussion.
